pkg/mdformatter/mdgen/mdgen_test.go

```diff
@@ -8,12 +8,40 @@ import (
 	"context"
 	"io/ioutil"
 	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/bwplotka/mdox/pkg/mdformatter"
 	"github.com/efficientgo/tools/pkg/testutil"
 )
 
+const (
+	testDocWithCode = `package test
+
+	import (
+		"bytes"
+		"fmt"
+		"strings"
+		"unsafe"
+	
+		"github.com/sergi/go-diff/diffmatchpatch"
+	)
+	
+	type Diff struct {
+		diffs    []diffmatchpatch.Diff
+		aFn, bFn string
+	}
+	
+	func yoloString(b []byte) string {
+		return *((*string)(unsafe.Pointer(&b)))
+	}
+	
+	func CompareBytes(a []byte, aFn string, b []byte, bFn string) Diff {
+		return Compare(yoloString(a), aFn, yoloString(b), bFn)
+	}	
+	`
+)
+
 func TestFormat_FormatSingle_CodeBlockTransformer(t *testing.T) {
 	f := mdformatter.New(context.Background(), mdformatter.WithCodeBlockTransformer(NewCodeBlockTransformer()))
```

Comment:
Why not putting this in our e2e framework with `input` `output` markdown files? (: